packages/react-search-ui-views/src/__tests__/MultiCheckboxFacet.test.js

```diff
@@ -157,6 +174,37 @@ describe("determining selected option from values", () => {
     ).toBe(false);
   });
 
+  it("will correctly determine which of the options is selected when using range filters and only the name matches", () => {
+    const wrapper = shallow(
+      <MultiCheckboxFacet
+        {...params}
+        options={rangeOptionsTimeBased}
+        values={[
+          {
+            // The name won't match in this case because even though the user selected and shared "1000 ms ago", the from
+            // is actually different when they reload the page, since it's relative to the current time.
```

Comment:
I'm not sure I'm reading this comment correctly? I think I get the idea behind it (having had to write date-based tests before) but `the from is actually different from when they reload the page` is throwing me off - is the comment missing a word between `the` and `from`?

Reply:
Ah, after coming back to this after having reviewed the removeFilter test down the page - I get this now! It's referring to "the `from` key". I might suggest wrapping the word `from` in backticks to help indicate that it's referring to a var/key name.

Reply:
Ugh, yeah, my comments are sometimes not so elegant. I will update this.